wgsl/index.bs

```diff
@@ -992,6 +1008,26 @@ array_decoration
   </xmp>
 </div>
 
+<div class='example' heading='Access qualifier'>
+  <xmp>
+    # Storage buffers
+    var<storage> buf1 : [[access(read)]] Buffer;       # Can read, cannot write.
```

Comment:
How is the `Buffer` declared in this case? Does it have no access decorator? What happens if it's declared with a decorator?

Reply:
In this case Buffer would be declared without an access qualifier. If multiple access qualifiers are provided that should be a validation error.

Reply:
Buffer can no longer be declared with an accessor.